### DISCUSSION TITLE
Please generate a new release tarball (Debian packaging)

### DISCUSSION THREAD

**Discussion by shayandoust** [taocpp/json#81] [General]
Hello,

I have been packaging tao-json[1] for the Debian-Med team[2]. Additionally, I have also been packaging tao-config[3], however this seems to require the latest version (HEAD) of the repository and the last release is from 18 Feb 2018. Please generate a new release tarball (*.tar.xz). This will allow me to simply get the tarball and integrate it into the package.

Thanks,
Shayan Doust

[1]: https://salsa.debian.org/med-team/tao-json
[2]: https://www.debian.org/devel/debian-med/
[3]: https://salsa.debian.org/med-team/tao-config

**Comment by d-frey** [comment]
Both Colin and me didn't have as much time for our open-source activities in the last weeks as we wished. We hope to soon find some time to finish the PEGTL 3.0.0 and, after that, we will bring both taoJSON as well as taoCONFIG into a state that we can put at least some rc/beta tags on them.

**Comment by d-frey** [comment]
Yeah, that took way too long... but I just created a new beta.12 pre-release.

**Comment by avsej** [comment]
@ColinH, how about new release? GCC 15 is getting rolled out (Fedora Rawhide at least), and ciso646 header has been removed there, breaking builds with taocpp/json. The issue has been fixed in PEGTL long time ago: https://github.com/taocpp/PEGTL/issues/319

**Comment by ColinH** [comment]
Apologies for the late reply, I'll try to find some time to fix this ASAP, at least to the point of not breaking the build.
